### ISSUE TITLE
App Responsiveness... Bug

### ISSUE THREAD

**Issue by BallisticPain** [ealves/protractor-recorder#46]
I was trying to figure out where to input the baseUrl and such. Thankfully I went to the animated gif and noticed I was missing the whole left "menu" of the page. I widened my window, and viola there's what I was looking for.

Here, it's out at 1280px width, and anything not as wide as that it disappears and I couldn't figure out how to access it...
![screen shot 2017-02-18 at 2 29 16 am](https://cloud.githubusercontent.com/assets/796026/23091728/32b7af38-f582-11e6-8d7d-b5e03ac7fd06.png)

Here's a screenshot at 1279px width
![screen shot 2017-02-18 at 2 31 11 am](https://cloud.githubusercontent.com/assets/796026/23091736/58a40f66-f582-11e6-8faf-7d527f8238ea.png)

Hopefully this helps someone else.

**Comment by phamdt** [comment]
related to #24

**Comment by guy-roberts** [comment]
Yep, this took me an hour to work out too, I almost didn't use this useful project because of it
